stock_move_entries/model/stock_move_entries.py

```diff
@@ -24,128 +25,40 @@
 #
 ##############################################################################
 from openerp.osv import fields, osv
-from openerp.tools.translate import _
 
 
-class account_move_line(osv.Model):
+class AccountMoveLine(osv.Model):
     _inherit = "account.move.line"
 
     """
     """
 
     _columns = {
-        'sm_id': fields.many2one('stock.move', 'Stock move ID'),
-        'location_id': fields.related('sm_id', 'location_id', string='Source Location',
-            type='many2one', relation='stock.location', store=True, help='Location Move Source'),
-        'location_dest_id': fields.related('sm_id', 'location_dest_id',
-            type='many2one', string='Destination Location', relation='stock.location', store=True,
-            help="Location Move Destination")
+        'sm_id': fields.many2one('stock.move', 'Stock move'),
     }
 
 
-class account_move(osv.Model):
-    _inherit = "account.move"
+class StockMove(osv.Model):
+    _inherit = "stock.move"
 
     """
     """
 
-    def _get_sm(self, cr, uid, ids, field_name, args, context=None):
-        res = {}
-        for record_id in ids:
-            cr.execute(
-                '''SELECT move_id,
-                          sm_id
-                   FROM account_move_line
-                   WHERE move_id = %s LIMIT 1 ''' % (record_id,))
-            if cr.rowcount:
-                record_id, sm_id = cr.fetchall()[0]
-                res[record_id] = sm_id
-            else:
-                res[record_id] = False
-        return res
-
     _columns = {
-        'sm_id': fields.function(_get_sm, method=True, type='many2one',
-                                 relation='stock.move',
-                                 string='Stock move ID', store=True),
+        'am_id': fields.one2many(
```

Comment:
Let us change the name of this field from am_id to aml_ids,
to adhere to established convention of using _ids for X2many fields.